Op 1559 upgrade

Pana opened the pull request:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/Conflux-Chain/js-conflux-sdk/271)
<!-- Reviewable:end -->

Pana committed:
add comment for new added fields

Pana committed:
misc: update changelog, opcode

Pana committed:
bump version

Pana committed:
fix lint

Pana committed:
new updates

Pana committed:
fix eslint broken

Pana committed:
add comment for feeHistory struct

Pana committed:
check

Pana committed:
remove unused option

wangdayong228 reviewed (approved):
> Reviewed 7 of 8 files at r1, 2 of 2 files at r2, 3 of 3 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/Conflux-Chain/js-conflux-sdk/271)* status: <span>:shipit:</span> complete! all files reviewed, all discussions resolved (waiting on @Pana)



<!-- Sent from Reviewable.io -->

Pana committed:
update maxFeePerGas calculate logic

change feeHistory field name

fix getFeeBurnt parameter issue

fix raw rpc error

fix lint

op code

update version
